### ISSUE TITLE
10b Step1 precludes off-screen headings to assist screen reader users

### ISSUE THREAD

**Issue by mraccess77** [Section508Coordinators/TrustedTester#9]
Step 1 in Test 10b fails heading that are not on-screen but are put in to assist screen readers understand content - was this intended?

If ALL of the following are TRUE, then the content PASSES:

1. Each programmatically determinable heading is serving as a visual heading on the page, AND
2. Each visual heading is programmatically defined.
